history/characters/10000_orc.txt

```diff
@@ -179,6 +180,26 @@
 	}
 	591.6.18={ death=yes }
 }
+bleedinghollow1={
+	name=Ugannar			
```

Comment:
Is it lore character?

Reply:
He is a placeholder for the title after Kil'rogg dies, as there is a "hole in history" title error otherwise. I'm thinking about creating an issue for this kind of problem. Since CK3 relies on having visible provinces as holdings on the map, there are quite a few characters (lore ones) in the history from the CK2 port who have titles with no land (e.g. d_wintergrasp for 190878) that are in the game but have no location and can't be interacted with. 

This is or can be the fate of tribal/adventurer nations who are defeated in lore history. The many Orc clans and Troll tribes with titles such as Bleeding Hollow, Shattered Hand or Stormreaver illustrate this. Although we can simply ignore these errors in the log, there is an opportunity for us to explore the intricates of Azeroth at level that Blizzard glossed over. Many Orcish clans were absorbed into Thrall's Horde and lost their cultural identity in the official canon, but I think it would be cool if we could ensure the continuity of these clans even as symbolic through a system like the [Zandalari House of the Fallen Tribes](https://wowpedia.fandom.com/wiki/House_of_the_Fallen_Tribes). In doing so, we allow these famous titles/dynasties to persist and potentially remain politically relevant through scheming into power again. The culture update in Royal Court futher enriches the value in retaining the cultural uniqueness of these titular nations. 

Anyway, we can discuss this more in a separate issue. For now, I conceptualise all these d_duchy title holders (they have no de jure lands) holding castle baronies (can be inherited) in a single county owned by the top liege of the realm. 

Thoughts @Remerod, @arithon, @Gaben1025?

Reply:
Give him some land if you want to keep him around Thrall.

Reply:
Leaving him as is for now, since this fix is targeted towards reducing the volume of the error log and lorewise the Bleeding Hollow only continued with some semblance of orgainisation in Outland. We can address him and other "loose-end" clans in #922.